rollrus.go

```diff
@@ -26,20 +26,47 @@ var wellKnownErrorFields = []string{
 // May be used as a rollbar client itself
 type Hook struct {
 	roll.Client
-	triggers []log.Level
+	triggers      []log.Level
+	ignoredErrors map[error]bool
```

Comment:
What does true and false mean?

If you want to check for existence do it like so:

```go
if _, ok := map[thing]; ok {
    // ....
}
```